src/index.js

```diff
@@ -3,12 +3,18 @@ import ReactDOM from 'react-dom/client';
 import { Provider } from 'react-redux';
 import App from './App';
 import store from './redux/configureStore';
+import { showRockets } from './redux/rockets/action-makers';
+import { BrowserRouter as Router } from "react-router-dom";
+
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <React.StrictMode>
     <Provider store={store}>
-      <App />
+      <Router>
+        <App />
+      </Router>
     </Provider>
-  </React.StrictMode>,
+  </React.StrictMode>
 );
+ console.log(store.dispatch(showRockets))
```

Comment:
Would like to suggest adding ```store.dispatch(showRockets())```  in the configureStore.js;  👍🏽

Reply:
I have removed the console log it was just  for testing